proxy/src/coco/peer.rs

```diff
@@ -106,6 +124,71 @@ pub fn list_projects(peer: &PeerApi) -> Result<Vec<Project>, error::Error> {
         .collect()
 }
 
+/// Get all peer IDs and their branches.
+///
+/// # Errors
+///
+///   * [`error::Error::LibradLock`]
+///   * [`error::Error::Git`]
+pub fn revisions(
+    peer: &PeerApi,
+    owner: &User,
```

Comment:
It took me a while to realize this was the current user. Could we document the function parameters?